contrib/pai_vscode/README.md

```diff
@@ -41,6 +41,18 @@ The extension has a useful feature called "Simulate Job Running". It enables the
 
 ![](https://raw.githubusercontent.com/Microsoft/pai/master/contrib/pai_vscode/assets/simulate-job.gif)
 
+## Sidebar
+HDFS Explorer and Job List view will be shown in the extension's view container (sidebar).
+
+### HDFS Explorer
+1. Connected to a PAI cluster's hdfs by double click the corresponding tree node.
+2. The cluster's directory structure will be shown in the tree view. Right click the folder/file node and perform selected file system operations.
+
+### Job List
+PAI cluster's latest jobs and jobs recently submitted by this extension will be shown in the job list tree view.
+
+- The job list will be auto refreshed every 10 seconds by default. 
```

Comment:
3. Job list will auto refresh every 10 seconds.
//is the default value changable?